### PR TITLE
Make TensorFlow follow some of its own deprecation warnings

### PR DESCRIPTION
**Pull request by girving**
TensorFlow should respect the deprecation warnings of TensorFlow.

### CODE REVIEW

**Comment by rthadur** [comment]
@girving please resolve conflicts

**Comment by girving** [comment]
@rthadur Test failures seem unrelated.  What's the right next step?

**Comment by rthadur** [comment]
@gunan can you help review this PR, if you are not the correct reviewer , can you please redirect, thank you

File: tensorflow/python/framework/func_graph.py
Comment (annarev) [inline tensorflow/python/framework/func_graph.py]: compute_device should be passed with as keyword argument now since create_op still has compute_shapes argument before it.

Comment (girving) [reply tensorflow/python/framework/func_graph.py]: Fixed.

**Review by annarev — APPROVED** [review]
Thank you for the clean up!

**Comment by girving** [comment]
@annarev Looks like `copybara` is failing.  Something I need to fix on my end?

**Comment by annarev** [comment]
I think it might be just waiting for approval (I will approve it soon).

**Pull request merged**
